projects/client/RabbitMQ.Client/RabbitMQ.Client.csproj

```diff
@@ -1,43 +1,49 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-
   <PropertyGroup>
     <VersionPrefix>6.0.0</VersionPrefix>
     <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
     <NoWarn>$(NoWarn);CS1591</NoWarn>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <AssemblyTitle>RabbitMQ Client Library for .NET</AssemblyTitle>
+    <Authors>VMware</Authors>
+    <Company>VMware, Inc. or its affiliates.</Company>
+    <Copyright>Copyright © 2007-2020 VMware, Inc. or its affiliates.</Copyright>
+    <Description>The RabbitMQ .NET client is the official client library for C# (and, implicitly, other .NET languages)</Description>
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <PackageTags>RabbitMQ</PackageTags>
     <PackageIcon>icon.png</PackageIcon>
-    <PackageProjectUrl>https://www.rabbitmq.com/dotnet.html</PackageProjectUrl>
     <PackageLicense>https://www.rabbitmq.com/dotnet.html</PackageLicense>
-    <AssemblyTitle>RabbitMQ Client Library for .NET</AssemblyTitle>
-    <Description>The RabbitMQ .NET client is the official client library for C# (and, implicitly, other .NET languages)</Description>
-    <Company>VMware, Inc. or its affiliates.</Company>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <PackageProjectUrl>https://www.rabbitmq.com/dotnet.html</PackageProjectUrl>
+    <PackageTags>rabbitmq, amqp</PackageTags>
     <Product>RabbitMQ</Product>
-    <Copyright>Copyright © 2007-2020 VMware, Inc. or its affiliates.</Copyright>
-    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <RepositoryUrl>https://github.com/rabbitmq/rabbitmq-dotnet-client</RepositoryUrl>
+    <RepositoryType>Git</RepositoryType>
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <AssemblyOriginatorKeyFile>../rabbit.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
+    <MinVerTagPrefix>v</MinVerTagPrefix>
+    <MinVerVerbosity>detailed</MinVerVerbosity>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+    <Copyright>Copyright © 2007-2020 VMware, Inc. or its affiliates.</Copyright>
   </PropertyGroup>
-
   <ItemGroup>
     <Compile Include="..\..\..\gensrc\autogenerated-api-0-9-1.cs" />
-  </ItemGroup>
-
-  <ItemGroup>
     <None Remove="icon.png" />
     <Content Include="icon.png" PackagePath="" />
-  </ItemGroup>
-
-  <ItemGroup>
+    <AssemblyAttribute Include="System.Runtime.CompilerServices.InternalsVisibleTo">
```

Comment:
This is how you have to do this now, but there is better syntax coming for this soon! I think it might arrive in the 3.1.200 SDK

Reply:
Yep I tried the new syntax which of course failed on 2.1 builds